cosmos/operators/airflow_async.py

```diff
@@ -60,13 +55,11 @@ class DbtSourceAirflowAsyncOperator(DbtBaseAirflowAsyncOperator, DbtSourceLocalO
     pass
 
 
-class DbtRunAirflowAsyncOperator(BigQueryInsertJobOperator):  # type: ignore
+class DbtRunAirflowAsyncOperator(BigQueryInsertJobOperator, DbtRunLocalOperator):  # type: ignore
 
     template_fields: Sequence[str] = (
         "full_refresh",
         "project_dir",
-        "gcp_project",
-        "dataset",
```

Comment:
Why did we remove these as templated fields?

Reply:
we have them back now